lib/osf-components/addon/components/registries/schema-block-renderer/editable/single-select-input/template.hbs

```diff
@@ -1,18 +1,23 @@
-<FormControls
-    @changeset={{@changeset}}
-    @disabled={{@disabled}}
-    @shouldShowMessages={{@shouldShowMessages}}
-    ...attributes
-    as |form|
->
-    <form.select
-        data-test-single-select-input
-        @id={{@uniqueID}}
-        @valuePath={{@schemaBlock.registrationResponseKey}}
-        @options={{this.optionBlockValues}}
-        @onchange={{@onInput}}
-        as |option|
-    >
-        {{option}}
-    </form.select>
-</FormControls>
+<div local-class='single-select-input-flexbox'>
```

Comment:
Benefits of this is it avoids more style.scss files. Downsides would be adding yet another layer of component whose only purpose is controlling the CSS. I would like to keep it as is unless someone has strong opinions about this.

Reply:
I don't think the problem is "more" files, it's the repetition within the files:
```css
.foo-input-flexbox {
    display: flex;
}

.input-box {
    flex: auto;
}
```
```hbs
<div local-class='input-flexbox'>
    <div local-class='input-box'>
        {{! ... }}
    </div>
    <Registries::SchemaBlockRenderer::HelperTextIcon @helpText={{this.schemaBlock.helpText}} />
</div>
```
If the above is the same for every block type (and we don't expect that to change), it would be nice to have it written only once, even if that means another component.

If there are special cases for specific block types, then this approach seems fine to me.